openqa-investigate

```diff
@@ -231,7 +231,7 @@ investigate() {
     # have more, ambiguous potential changes that we need to bisect on
 
     # sync by writing initial investigation comment (edited later)
-    comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id"); rc=$?
+    comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
```

Comment:
What if `rc` is still 255 from the previous call and `sync_via_investigation_comment` returns 0? Then we wouldn't override the previous 255 and continue instead of returning 0. But returning 0 is what this code should do then, right?

Reply:
good point! I need to reset rc

Reply:
#206